src/mlmd/LineageView.tsx

```diff
@@ -208,7 +208,7 @@ export class LineageView extends React.Component<LineageViewProps, LineageViewSt
       }
 
       currentCard.elements.push({
-        resource: artifact,
+        typedResource: { type: 'artifact', resource: artifact },
```

Comment:
If we're using auto-styles could we keep it matched to other styles in the codebase, ie:
```suggestion
        typedResource: {type: 'artifact', resource: artifact},
```

Reply:
If we use prettier, it's not possible. Prettier is very opinionated about the format, it was invented to stop everyone from arguing about format and just use the common one (and bear with whatever you don't like about it, to be honest, I like their formatting very much, so it's not really a problem for me).
https://prettier.io/docs/en/options.html

I think that saves engineer resources from my previous experience and also when working on kubeflow/pipelines UI.

Reply:
what do you think? I'm following your suggestion for this PR

Reply:
Yeah I agree with consistency, so if it's an automated tool, we can enforce those styles across the codebase. SGTM :)